catalog/src/main/AndroidManifest.xml

```diff
@@ -43,6 +43,15 @@
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" />
+                <data android:host="android.spark.adevinta.com" />
+            </intent-filter>
```

Comment:
I don't think this was added on purpose 🤔

Reply:
Ah actually it was but not directly but for a follow-up PR to add deep link support.
I'll readd it later